[DOCS] Revised glossary for GX 1.0 (#9714)
great-expectations/great_expectations: docs/docusaurus/docs/reference/learn/glossary.md

title: "Glossary"
---

> [Action](/core/validate_data/checkpoints/manage_checkpoints.md): A Python class with a `run` method that takes a Validation Result and does something based on its contents.

Reviewer: ```suggestion
[Action](/core/validate_data/checkpoints/manage_checkpoints.md): A component that integrates Great Expectations with other tools based on a Validation Result, like sending notifications based on the validation's outcomes.
```

> [Batch](/core/manage_and_access_data/manage_batches.md): A selection of records retrieved from a Data Asset according to the parameters passed to a Batch Definition.

Reviewer: Shouldn't these be pointing to the glossary pages? Or are you thinking that we're just going down to the 1 sentence versions exclusively?

Author: Since the existing glossary "learn" content was falling out of date, and a fair amount of the new 1.0 objects didn't have existing "learn" content or further changed how the existing objects interacted I left out the original learn topics and linked to the practicals for this draft.

I think that revising old and creating new learn topics for each of these subjects will probably mean sufficient lift to be put into its own epic as a concerted effort, and until then I'd prefer that we link to the up-to-date practical guides than risk overlooking content that should be revised by patching in the old docs and inadvertently sending people to outdated information.

> [Batch](/core/manage_and_access_data/manage_batches.md): A selection of records retrieved from a Data Asset according to the parameters passed to a Batch Definition.

Reviewer: ```suggestion
[Batch](/core/manage_and_access_data/manage_batches.md): A set of data that is validated as part of a single test run.
```

> [Batch Definition](/core/manage_and_access_data/manage_batch_definitions/manage_batch_definitions.md): A reference that tells GX how to slice the records in a Data Asset into subsets.

Reviewer: ```suggestion
[Batch Definition](/core/manage_and_access_data/manage_batch_definitions/manage_batch_definitions.md): A configuration for how a Data Asset should be divided into individual batches for testing.
```

[Checkpoint](/core/validate_data/checkpoints/manage_checkpoints.md): An object that groups Validation Definitions and runs them with shared parameters and automated Actions.  Checkpoints are the primary means for validating data in a production deployment of Great Expectations.

[Data Asset](/core/manage_and_access_data/manage_data_assets.md): A collection of records within a Data Source which are usually grouped based on the underlying data system and can be further sliced into Batches according to a desired specification.

> [Data Context](/core/installation_and_setup/manage_data_contexts.md): A storage location for metadata such as configurations for GX components and output such as Validation Results and the metrics associated with them.

Reviewer: ```suggestion
[Data Context](/core/installation_and_setup/manage_data_contexts.md): A Data Context stores all of the configuration for a project in Great Expectations, and provides an API to access and update it.
```

[Data Docs](/core/installation_and_setup/manage_data_docs.md): Human readable documentation generated from Great Expectations metadata detailing Expectations, Validation Results, etc.

[Data Source](/core/manage_and_access_data/manage_data_sources/manage_data_sources.md): An object that tells GX how to access a specific source of external data and provides a unified API within GX for organizing that data into Data Assets which can be retrieved. 

[Expectation](/core/create_expectations/expectations/manage_expectations.md): A verifiable assertion about data.

> [Expectation Suite](/core/create_expectations/expectation_suites/manage_expectation_suites.md): A group of Expectations that describe the same set of data.

Reviewer: ```suggestion
[Expectation Suite](/core/create_expectations/expectation_suites/manage_expectation_suites.md): A group of Expectations that describe how data should be tested.
```
